lib/assets.dart

```diff
@@ -7,4 +7,27 @@ class _Icons {
   String devcademy = 'assets/icons/devcademy.png';
 }
 
-class _Images {}
+class _Images {
+  String london = 'assets/popular_locations/London.png';
+  String barcelona = 'assets/popular_locations/Barcelona.png';
+  String tokyo = 'assets/popular_locations/Tokyo.png';
+  String ny = 'assets/popular_locations/NewYork.png';
+
+  String ap1 = 'assets/guests_love/ap1.png';
+  String ap2 = 'assets/guests_love/ap2.png';
+
+  String getImage(String name) {
```

Comment:
You can remove getImage function and add this class 
class Assets {
  static _Images images = _Images();
}
Then you will be able to call Assets.images.london